src/rating-summary/ratingSummary.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FC, ReactElement, useMemo } from 'react';
+import { ISummaryProp } from './types';
+import classes from './styles.module.scss';
+import starImg from '../assets/star-grey.svg';
+import { COMMA_SEPARATED_NUMBER_REGEX, Elements, RatingIds } from '../constants';
+
+const RatingSummary: FC<ISummaryProp> = (props) => {
+  const { ratings,
+    renderLabel,
+    showCount = true,
+    showAnimation = true,
+    styles={},
+    chartColors} =props;
+
+  const getRatingLabelFromRatingId = (ratingId: string): ReactElement => {
+    if (renderLabel) return (renderLabel(ratingId));
+    let label = '';
+    switch (ratingId) {
+    case RatingIds.one:
+      label = '1'
+      break;
+    case RatingIds.two:
+      label = '2'
+      break;
+    case RatingIds.three:
+      label = '3'
+      break;
+    case RatingIds.four:
+      label = '4'
+      break;
+    case RatingIds.five:
+      label = '5'
+      break;
+    default:
+    }
+    return (
+      <div
+        className={classes.label}
+        id={`${ratingId}-label`}
+      >
+        <img src={starImg} alt="" width="10" height="10" />
+        {label}
+      </div>
+    );
+  };
+
+  const formatToNumberWithCommas = (num: number): string => (num.toString().replace(COMMA_SEPARATED_NUMBER_REGEX, ","));
+
+  const getTotalRatingsCount = useMemo((): number => (
```

Comment:
Name gives an idea that this is a function, but it is a value

Reply:
changed to a variable name